docs/data/base/components/number-input/number-input.md

```diff
@@ -91,6 +91,82 @@ The following code snippet:
 />
 ```
 
+### Events
+
+The NumberInput component provides two props - `onChange` and `onInputChange` - that accept event handlers for when the value of the component changes.
+
+#### onChange
+
+`onChange` accepts a custom event handler that is called with two arguments - the underlying event, and the latest "clamped" value:
+
+```ts
+onChange: (
+  event: React.FocusEvent<HTMLInputElement> | React.PointerEvent | React.KeyboardEvent,
+  value: number | undefined,
+) => void;
+```
+
+It's called when the `<input>` element is blurred, or when the stepper buttons are clicked, after the value has been clamped based on the min, max, or step props.
```

Comment:
```suggestion
It's called when the `<input>` element is blurred, or when the stepper buttons are clicked, after the value has been clamped based on the `min`, `max`, or `step` props.
```

Reply:
we might want to link those prop names to the corresponding sections since this would be the first mention of them on this page if it stays up here under `Component`